cross-modal-search/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:1.2.0
+FROM jinaai/jina:2.0.8-perf
```

Comment:
@Yongxuanzhang  Version is now bumped to 2.0.8. However, we are seeing an issue happening when running cross-modal-search with docker, detailed [here](https://github.com/jina-ai/examples/pull/670#issuecomment-883190942). This needs to be further investigated. Thanks.